defaults/main.yml

```diff
@@ -3,16 +3,18 @@
 # Enable or disable PKI support
 pki: True
 
+# Default domain
+pki_domain: '{{ ansible_domain }}'
```

Comment:
If it works with `ansible_domain` empty then I guess we're OK. There's no difference between `default("")` and `d("")`, the latter is just shorter.